packages/manager/apps/pci-private-registry/src/components/CIDR/AuthorizationLabel.component.tsx

```diff
@@ -19,50 +19,52 @@ import { useTranslation, Trans } from 'react-i18next';
 const AuthorizationLabel = () => {
   const { t } = useTranslation(['ip-restrictions']);
   return (
-    <OsdsPopover>
-      <span slot="popover-trigger">
-        <OsdsText
-          size={ODS_TEXT_SIZE._500}
-          hue={ODS_TEXT_COLOR_HUE._800}
-          level={ODS_TEXT_LEVEL.body}
-          color={ODS_TEXT_COLOR_INTENT.primary}
-        >
-          {t('private_registry_cidr_authorization')}
-        </OsdsText>
-
-        <OsdsIcon
-          name={ODS_ICON_NAME.HELP}
-          size={ODS_ICON_SIZE.xs}
-          className="ml-4 cursor-help"
-          color={ODS_THEME_COLOR_INTENT.primary}
-        />
-      </span>
-
-      <OsdsPopoverContent>
-        <Trans ns="ip-restrictions">
-          <OsdsText
-            color={ODS_THEME_COLOR_INTENT.text}
-            level={ODS_TEXT_LEVEL.body}
-          >
-            {t('private_registry_cidr_help_authorized_component')}
-          </OsdsText>
-          <br />
+    <div className="whitespace-normal inline">
+      <OsdsPopover>
+        <span slot="popover-trigger">
           <OsdsText
-            color={ODS_THEME_COLOR_INTENT.text}
+            size={ODS_TEXT_SIZE._500}
+            hue={ODS_TEXT_COLOR_HUE._800}
             level={ODS_TEXT_LEVEL.body}
+            color={ODS_TEXT_COLOR_INTENT.primary}
           >
-            {t('private_registry_cidr_help_authorized_component_part2')}
+            {t('private_registry_cidr_authorization')}
           </OsdsText>
-          <br />
-          <OsdsText
-            color={ODS_THEME_COLOR_INTENT.text}
-            level={ODS_TEXT_LEVEL.body}
-          >
-            {t('private_registry_cidr_help_authorized_component_part3')}
-          </OsdsText>
-        </Trans>
-      </OsdsPopoverContent>
-    </OsdsPopover>
+
+          <OsdsIcon
+            name={ODS_ICON_NAME.HELP}
+            size={ODS_ICON_SIZE.xs}
+            className="ml-4 cursor-help"
+            color={ODS_THEME_COLOR_INTENT.primary}
+          />
+        </span>
+
+        <OsdsPopoverContent>
+          <Trans ns="ip-restrictions">
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+            >
+              {t('private_registry_cidr_help_authorized_component')}
+            </OsdsText>
+            <br />
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+            >
+              {t('private_registry_cidr_help_authorized_component_part2')}
+            </OsdsText>
+            <br />
+            <OsdsText
+              color={ODS_THEME_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+            >
+              {t('private_registry_cidr_help_authorized_component_part3')}
+            </OsdsText>
+          </Trans>
```

Comment:
If we just want add spacing between element. Maybe use container with gap (`flex flex-col gap-2`) ?

Allow to have better control ?